src/lake/Lake/Util/Name.lean

```diff
@@ -25,6 +38,9 @@ abbrev OrdNameMap α := RBArray Name α Name.quickCmp
 @[inline] def OrdNameMap.empty : OrdNameMap α := RBArray.empty
 @[inline] def mkOrdNameMap (α : Type) : OrdNameMap α := RBArray.empty
 
+instance [ToJson α] : ToJson (OrdNameMap α) where
+  toJson m := .obj <| m.toRBMap.revFold (fun ps n s => ps.insert compare n.toString (toJson s)) ∅
```

Comment:
Note this does not actually preserve the order, which is important, hence why a `OrdNameMap` not a `NameMap` is used here.

Reply:
I don't see any way to both preserve the order and keep the keys with this interface

Reply:
I fixed this by just removing the instance, because the only place it was being used was for maps of lakeLibs and lakeExes and these already have the name in them.